pkg/didcomm/protocol/didexchange/states_test.go

```diff
@@ -1776,6 +1854,26 @@ func TestGetVerKey(t *testing.T) {
 		require.Equal(t, publicDID.Service[0].RecipientKeys[0], result)
 	})
 
+	t.Run("returns verkey from implicit (interop) oob invitation", func(t *testing.T) {
+		publicDID, err := diddoc.ParseDocument([]byte(sovDoc))
+		require.NoError(t, err)
+		invitation := newOOBInvite(publicDID.ID)
+		ctx.connectionRecorder = connRecorder(t, testProvider())
+		ctx.vdRegistry = &mockvdr.MockVDRegistry{
+			ResolveValue: publicDID,
+		}
+		ctx.doACAPyInterop = true
+
+		err = ctx.connectionRecorder.SaveInvitation(invitation.ThreadID, invitation)
+		require.NoError(t, err)
+
+		result, err := ctx.getVerKey(invitation.ThreadID)
+		require.NoError(t, err)
+		require.Equal(t, publicDID.Service[0].RecipientKeys[0], result)
```

Comment:
This Equal() call means both expected and actual results are the same object reference. Usually it's better to use EqualValues if you're looking for content equality.

Reply:
Those are strings, `Equal()` is fine here :)